src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -94,13 +93,12 @@ void LoadARFF(const std::string& filename,
       else
       {
         throw std::runtime_error("unknown ARFF annotation '" + (*tok.begin()) +
-            "'");
+                                 "'");
```

Comment:
There is an option for this `AlignAfterOpenBracket` in the style guide. Three possible values are: 

1. `Align`: Align parameters on the open bracket, e.g.:
   ```c++
   someLongFunction(argument1,
                    argument2);
   ```
2. `DontAlign`:  Don’t align, instead use 
   ContinuationIndentWidth, e.g.:
   ```c++
   someLongFunction(argument1,
       argument2);
   ```
3. `AlwaysBreak`: Always break after an open bracket, if the parameters don’t fit on a single line, e.g.:
   ```c++
   someLongFunction(
       argument1, argument2);
   ```

I can handle this by setting this configuration option `AlignAfterOpenBracket` to `DontAlign` but it will change the alignment in constructor initializer lists and function declarations too with 4 indent width. Should I go ahead with this?